bcgov/HOUS-permit-portal
# Feature/bphh 673/building characteristics summary

## Description

Adds support for the "Building Characteristics Summary" section of the step code checklist.

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

Implements [BPHH-673](https://hous-bssb.atlassian.net/browse/BPHH-673).

## Steps to QA

1. In permit application form, navigate to energy step code section and open the compliance tool
2. Scroll to section D: Building Characteristics Summary 
3. Inputs should be present and user should be able to populate inputs / selects and update the checklist. Changes should save and be reflected in the UI.

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?
